tutorials-v5/time-evolution/022_v5_paper-smesolve.md

```diff
@@ -0,0 +1,132 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: qutip-tutorials-v5
+    language: python
+    name: python3
+---
+
+# QuTiPv5 Paper Example: Stochastic Solver - Homodyne Detection
+
+Authors: Maximilian Meyer-Mölleringhof (m.meyermoelleringhof@gmail.com)
+
+## Introduction
+
+When modelling an open quantum system, stochastic noise can be used to simulate a large range of phenomena.
+In the `smesolve()` solver, noise appears because of continuous measurement.
+It allows us to generate the trajectory evolution of a quantum system conditioned on a noisy measurement record.
+Historically speaking, such models were used by the quantum optics community to model homodyne and heterodyne detection of light emitted from a cavity.
+However, this solver is of course quite general and can thus also be applied to other problems.
+
+In this example we look at an optical cavity whose output is subject to homodyne detection.
+Such a cavity obeys the general stochastic master equation
+
+$d \rho(t) = -i [H, \rho(t)] dt + \mathcal{D}[a] \rho (t) dt + \mathcal{H}[a] \rho dW(t)$
+
+with the Hamiltonian
+
+$H = \Delta a^\dag a$
```

Comment:
`\dag` is not defined

Reply:
Just opening the file in jupyter notebook (with jupytext), looks like this for me:

![image](https://github.com/user-attachments/assets/d51227c3-5852-4d1f-bc63-f7dfb92a7507)

Reply:
Alright, thanks. Today I learned, VS Code and Jupter Lab use different renderers to display Latex. Should be fixed now.